databricks/koalas/tests/test_dataframe.py

```diff
@@ -1750,6 +1751,115 @@ def test_binary_operators(self):
+        # pdf = pd.DataFrame({"a": ["x"], "b": ["y"]})
+        # self.assertRaises(TypeError, lambda: pdf["a"] - pdf["b"])
+
+    def test_binary_operator_truediv(self):
+        kdf = ks.DataFrame({"a": ["x"], "b": [1]})
+        ks_err_msg = re.escape("division can not be applied on string series or literals")
+
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["a"] / kdf["b"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["b"] / kdf["a"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["b"] / "literal")
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: "literal" / kdf["b"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: 1 / kdf["a"])
+
+        # pdf = pd.DataFrame({"a": ["x"], "b": [1]})
+        # self.assertRaises(TypeError, lambda: pdf["a"] / pdf["b"])
+        # self.assertRaises(TypeError, lambda: pdf["b"] / pdf["a"])
+        # self.assertRaises(TypeError, lambda: "literal" / pdf["b"])
+        # self.assertRaises(TypeError, lambda: 1 / pdf["a"])
+
+    def test_binary_operator_floordiv(self):
+        kdf = ks.DataFrame({"a": ["x"], "b": [1]})
+        ks_err_msg = re.escape("division can not be applied on string series or literals")
+
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["a"] // kdf["b"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["b"] // kdf["a"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["b"] // "literal")
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: "literal" // kdf["b"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: 1 // kdf["a"])
+
+        # pdf = pd.DataFrame({"a": ["x"], "b": [1]})
+        # self.assertRaises(TypeError, lambda: pdf["a"] // pdf["b"])
+        # self.assertRaises(TypeError, lambda: pdf["b"] // pdf["a"])
+        # self.assertRaises(TypeError, lambda: "literal" // pdf["b"])
+        # self.assertRaises(TypeError, lambda: 1 // pdf["a"])
+
+    def test_binary_operator_mod(self):
+        kdf = ks.DataFrame({"a": ["x"], "b": [1]})
+        ks_err_msg = re.escape("modulo can not be applied on string series or literals")
+
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["a"] % kdf["b"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["b"] % kdf["a"])
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: kdf["b"] % "literal")
+        self.assertRaisesRegex(TypeError, ks_err_msg, lambda: 1 % kdf["a"])
+
+        # pdf = pd.DataFrame({"a": ["x"], "b": [1]})
+        # self.assertRaises(TypeError, lambda: pdf["a"] % pdf["b"])
+        # self.assertRaises(TypeError, lambda: pdf["b"] % pdf["a"])
+        # self.assertRaises(TypeError, lambda: 1 % pdf["a"])
+
+    def test_binary_operator_multiply(self):
+        kdf = ks.DataFrame({"a": ["x", "y"], "b": [1, 2]})
+        pdf = pd.DataFrame({"a": ["x", "y"], "b": [1, 2]})
```

Comment:
Shall we use `ks.from_pandas()`?

```py
pdf = pd.DataFrame({"a": ["x", "y"], "b": [1, 2]})
kdf = ks.from_pandas(pdf)
```

Reply:
Good idea! Updated all usage in new tests added.